apps/accounts/models/automations/push_account.py

```diff
@@ -23,7 +23,8 @@ def create_nonlocal_accounts(self, usernames, asset):
         create_usernames = set(usernames) - set(account_usernames)
         create_account_objs = [
             Account(
-                name=f'{username}-{secret_type}', username=username,
+                name=f"{username}-{secret_type}" if secret_type != SecretType.PASSWORD else username,
+                username=username,
                 secret_type=secret_type, asset=asset,
             )
             for username in create_usernames
```

Comment:
Please provide the code you want to check and I will analyze it for any inconsistencies or improvements.